Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ProjectController.cs

```diff
@@ -140,52 +135,44 @@ public class ProjectController : Controller
       * @apiSuccessExample {json} Success-Response:
       * HTTP/1.1 200 OK 
       * {
-      *     "description":"Add Project in ProjectTable"
+      *     "description":"Add new project"
       * }
       */
         [Authorize]
         [HttpPost]
-        [Route("addProject")]
+        [Route("")]
         public async Task<IActionResult> addProject([FromBody]ProjectAc project)
         {
-            try
+            var createdBy = _userManager.GetUserId(User);
+            if (ModelState.IsValid)
             {
-                var createdBy = _userManager.GetUserId(User);
-                if (ModelState.IsValid)
+                ProjectAc projectAc = _projectRepository.checkDuplicate(project);
+                if (projectAc.Name != null && projectAc.SlackChannelName != null)
```

Comment:
We check the string for nullability using String.IsNullOrEmpty not by comparing with null. Why are you doing additional check since you have already checked for ModelState.IsValid?